### PR TITLE
AISDK-156: Add support for human transcription

### PR DESCRIPTION
**Pull request by k-weng**

### CODE REVIEW

File: src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java [hunk 1]
```diff
@@ -145,7 +183,7 @@ public Double getDurationSeconds() {
   /**
    * Sets the audio duration.
    *
-   * @param durationSeconds An Integer value to set as audio duration.
+   * @param durationSeconds An Double value to set as audio duration.
```
Comment (amikofalvy) [inline src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java]:
```suggestion
   * @param durationSeconds A Double value to set as audio duration.
```

File: src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java [hunk 2]
```diff
@@ -145,7 +183,7 @@ public Double getDurationSeconds() {
   /**
    * Sets the audio duration.
    *
-   * @param durationSeconds An Integer value to set as audio duration.
+   * @param durationSeconds A Double value to set as audio duration.
```
Comment (amikofalvy) [inline src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java:186]: I don't think it is necessary to document type in the JavaDoc.

Comment (amikofalvy) [reply src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java:186]: We can avoid this type of change if we don't replicate the typing in the javadoc.

Comment (k-weng) [reply src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJob.java:186]: Was following the established patterns that we had for our other params

**Review by amikofalvy — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -69,6 +69,21 @@ try {
...
 
+You can also submit a job to be handled by a human transcriber using our [Human Transcription](https://www.rev.ai/docs#section/Human-Transcription-(Labs)) option.
+```
+String urlLinkToFile = "https://www.rev.ai/FTC_Sample_1.mp3";
+RevAiJobOptions options = new RevAiJobOptions();
+
+// set to perform human transcription
+options.setTranscriber("human");
+
+// optional job options
```
Comment (kbridbur) [inline README.md:80]: Should segments to transcribe also be included here? The segment from 0 to 2 minutes should work with that file

Comment (kbridbur) [reply README.md:80]: Potentially test mode as well

**Review by kbridbur — APPROVED** [review]
Two small questions. Looks good otherwise

File: README.md (hunk 3 above)
Comment (k-weng) [reply README.md:80]: Done

**Pull request merged**